examples/complete/versions.tf

```diff
@@ -0,0 +1 @@
+versions.tf
```

Comment:
This is incorrect. Please copy and paste the versions.tf in the repo root to this directory.

Reply:
My bad, let me fix it

Reply:
No worries, I fixed it.